sway-book/src/documentation/language/control-flow/match/index.md

```diff
@@ -0,0 +1,13 @@
+# match
+
+[If expressions](if-expressions.md) can be used to check a large number of conditions however there is an alternative syntax which allows us to perform advanced pattern matching.
```

Comment:
```suggestion
[If expressions](if-expressions.md) can be used to check a large number of conditions. However, there is an alternative syntax which allows us to perform advanced pattern matching.
```

Reply:
As above, technically the only correct punctuation of "however" as a coordinating adverb is  

> ... ; however, ...